app/src/views/components/friends/friend_request_item.tsx

```diff
@@ -0,0 +1,127 @@
+import { Image, StyleSheet, View } from "react-native";
+import { ThemedText } from "@/src/views/components/theme/themed_text";
+import { ThemedView } from "@/src/views/components/theme/themed_view";
+import { Colors } from "@/constants/Colors";
+import { ThemedIconButton } from "@/src/views/components/theme/themed_icon_button";
+import { useFriendIconViewModel } from "@/src/viewmodels/components/friends/FriendIconViewModel";
+
+/**
+ * Friend Request Item component
+ * @param name : name of the user
+ * @param avatar : avatar of the user
+ * @param onAccept : function to accept a friend request
+ * @param onDecline : function to decline a friend request
+ * @returns FriendRequestItem Component
+ */
+export function FriendRequestItem({
+  name,
+  avatar,
+  onAccept,
+  onDecline,
+  testID,
+}: any) {
+  const { firstLetter } = useFriendIconViewModel({ name });
+
+  return (
+    <ThemedView style={styles.requestItem} testID="friend-request-item">
+      {avatar ? (
```

Comment:
You should surrender the avatar by a Boolean because it will be causing issues on sonarcloud.